addons/mail/static/src/js/models/mail_failure.js

```diff
@@ -35,6 +36,7 @@ var MailFailure = Class.extend(Mixins.EventDispatcherMixin, ServicesMixin, {
         this._moduleIcon = data.module_icon;
         this._notifications = data.notifications;
         this._recordName = data.record_name;
+        this._failureType = data.failure_type;
```

Comment:
alphabetical order of attributes: `_failureType` between `_documentModel` and `_lastMessageDate`

Reply:
```js
this._failureType = data.failure_type || 'mail';
```